src/bin/playground.cpp

```diff
@@ -408,7 +417,55 @@ file_header read_file_header(std::string filename) {
   return file_header;
 }
 
+void prepare_filestream(const char* fname) {
```

Comment:
even if we probably should remove this function, for future purposes using std::string is cpp-style compared to const char* which is C-style (and as Tobi is happy to remind you `fname` is not an allowed variable name according to the style guide)